editor/src/components/assets.ts

```diff
@@ -68,15 +72,36 @@ export function projectContentFile(
   fullPath: string,
   content: UIJSFile | CodeFile | ImageFile | AssetFile,
 ): ProjectContentFile {
+  switch (content.type) {
+    case 'CODE_FILE':
+    case 'ASSET_FILE':
+    case 'IMAGE_FILE':
+    case 'UI_JS_FILE':
+      break
+    default:
+      throw new Error(`Not a file: ${JSON.stringify(content)}`)
```

Comment:
Maybe worth adding the exhaustive check here

Reply:
Actually I think this can go now, I used it to identify a bug.

Reply:
Fixed in 477c7b28.